src/Apps/Order2/Routes/Checkout/CheckoutContext/Order2CheckoutContext.tsx

```diff
@@ -443,6 +480,71 @@ const reducer = (state: CheckoutState, action: Action): CheckoutState => {
           return [...acc, current]
         }, [] as CheckoutStep[]),
       }
+    case "EDIT_PAYMENT":
+      return {
+        ...state,
+        steps: state.steps.reduce((acc, current) => {
```

Comment:
#non-blocking. Started to see repetition for edit actions. Probably can DRY separately later.

Reply:
I agree, just don't know what the best way to manage these step states is long term yet.